File: java-lhdiff-project/datasets/Aakanksha/File08_New.java
// File08_New.java - Mixed Complex Changes
public class File08 {
    private int counter = 0;

    public void resetAndLog(int initialValue) { // Moved method
        this.counter = initialValue;
        // Line split
        String logMessage = "Counter reset to ";
        logMessage += initialValue + ". Logged at " + System.currentTimeMillis();
        System.out.println(logMessage);
    }

    public void increment() { // Moved method
        counter++;
        String currentCount = "Current count: " + counter; // Line merged
        System.out.println(currentCount); // Line merged
    }
}

File: python-lhdiff-project/datasets/Aakanksha/File08_New.py
# File08_New.py - Mixed Complex Changes

from time import time

class File08:
    def __init__(self):
        self.counter = 0

    def reset_and_log(self, initial_value):  # Moved method
        self.counter = initial_value
        # Line split
        log_message = "Counter reset to "
        log_message += f"{initial_value}. Logged at {int(time.time() * 1000)}"
        print(log_message)

    def increment(self):  # Moved method
        self.counter += 1
        current_count = f"Current count: {self.counter}"  # Line merged
        print(current_count)  # Line merged

    def decrement(self):  # Added method
        self.counter -= 1
        current_count = f"Current count: {self.counter}"
        print(current_count) 